common/views/themes/config.ts

```diff
@@ -146,6 +146,7 @@ export const themeValues = {
 
   // Keyboard focus uses a hard box shadow of 0.7 opacity 'turquoise'
   focusBoxShadow: '0 0 0 3px rgba(92, 184, 191, 0.7)',
+  headerMediumBreakpoint: 825,
```

Comment:
Could we leave a note explaining this?

Reply:
I'm thinking it might be better to add it as a key on the `sizes` property so that it can be used with the `respondBetween` mixin. That way, it would also live in a place where it is clearer what it does. But a comment wouldn't hurt as well, yep.

Reply:
Great idea.

Reply:
…added a `tweakpoints` key in the end to accomplish this, since we were already generating some classes based on the `sizes` and I didn't want to spit out any more classes than necessary there.